src/styles/theme.scss

```
@@ -34,8 +35,10 @@ $spacingLarge: 24px;
 $layersDrawer: 4;
 $baseMap: 1;
 
-/** Header height **/
+/** Component dimensions **/
 $headerHeight: 54px;
+$regionSelectWidth: 300px;
+$yearSelectHeight: 40px;
```

```suggestion
$selectButtonHeight: 40px;
```

And then also use that value to replace the hardcoded value in `LayersDrawer.module.scss` on line 11